features/authentication/contracts/logged_user.md

```diff
@@ -0,0 +1,16 @@
+# Logged user
+
+## Logged user [/auth/me]
+
+This resource represents the retrieval of user profile for the current logged suer.
+
+### Request password change [] [GET] 🔒
+
++ **Response 204** (application/json)
+  ```json
+  {
+    "name": "Foo",
+    "surname": "Bar",
+    "email": "foo@mrmilu.com",
```

Comment:
As long as accounts have to be validated, accounts have states and there should be a way to know in which state my account is.

Reply:
True.